src/color_wheel.cpp

```diff
@@ -571,4 +667,33 @@ void ColorWheel::dropEvent(QDropEvent* event)
     }
 }
 
+void ColorWheel::clearHarmonies()
+{
+    p->ring_editors.clear();
+    p->current_ring_editor = -1;
+    Q_EMIT harmonyChanged();
+    update();
+}
+
+void ColorWheel::addHarmony(double hue_diff, bool editable, int symmetric_to, int opposite_to)
```

Comment:
This doesn't emit `harmonyChanged`, is it intentional?

From what I understand you are supposed to call `addHarmony` a couple times and then `applyHarmonies`. If this is the case it should be documented in the comment for `addHarmony`.